src/main.cpp

```diff
@@ -47,10 +47,12 @@
 extern void qt_set_sequence_auto_mnemonic ( bool bEnable );
 #endif
 #include <memory>
-#include "rpcserver.h"
-#include "serverrpc.h"
-#ifndef SERVER_ONLY
-#    include "clientrpc.h"
+#ifndef NO_JSON_RPC
```

Comment:
As most occurences are doubly-negated (*if not defined no json rpc*), I wonder if it makes sense to turn the logic around (`#ifdef JSON_RPC`).

Reply:
It's consistent with "nosound" and was somewhat consistent with "headless" (which became "serveronly", which is more positive).

Also having it as "JSON_RPC" would imply an "off by default" state, rather than the "on by default" state we have.